modules/cluster-logging-systemd-scaling.adoc

```diff
@@ -71,7 +71,7 @@ include a unit: "year", "month", "week", "day", "h" or "m". Enter `0` to disable
 * `none` to not store logs. systemd drops all logs.
 <7> Specify the timeout before synchronizing journal files to disk for *ERR*, *WARNING*, *NOTICE*, *INFO*, and *DEBUG* logs.
 systemd immediately syncs after receiving a *CRIT*, *ALERT*, or *EMERG* log. The default is `1s`.
-<8> Specify the maximum size the journal can use. The default is `8g`.
+<8> Specify the maximum size the journal can use. The default is `8G`.
```

Comment:
Couldn't hurt to changes this to something like: "Specify the maximum size in <UNITS> that the journal can use. The default is `8G`."